sdk/python/tests/unit/permissions/auth/server/conftest.py

```diff
@@ -84,4 +167,7 @@ def _default_store(temp_dir, auth_config: str):
     assert result.returncode == 0
 
     fs = FeatureStore(repo_path=repo_path)
+
+    fs.apply(permissions)
```

Comment:
since permissions is `Optional`, can we check it'snot `None`?

Reply:
Will fix, its not optional anymore